make.inc.windows_msys

```diff
@@ -0,0 +1 @@
+MSYS=ON
```

Comment:
So, it looks like the Windows user has two routes now: mingw or msys. Both flags can't be set ON at the same time. Could you add something about how to use this in comments here, or in docs/devnotes.rst  ? Thanks.

Reply:
As in the above comments, windows user can use mingw gcc compiler. the compiler can be called in msys or wsl.
The commands 'copy' or 'move' in makefile matters when using msys or wsl. Adding comments in makefile.

Reply:
Ok, great - I'll wait for your comments in makefile, and docs in the devnotes.rst.

Reply:
Is the CI testing both MSYS and MINGW or just one? Would probably be a good idea to do both. If not, we should specify which one is tested (i.e. supported).